Engineering/ACID model.md

```diff
@@ -0,0 +1,36 @@
+---
+tags: engineering, database, transaction
+author: Nguyen Xuan Anh
+---
+
+# What is the ACID model?
```

Comment:
Can we avoid using h1 in all the files? it's because h1 will be automatically inserted from the file name.